packages/module-app-info/src/server/server.ts

```
@@ -41,6 +53,33 @@ export class SystemSettingsPlugin extends Plugin {
       };
     });
 
+    this.app.resourcer.define({
+      name: 'systemSettings',
+      actions: {
+        get: async (ctx, next) => {
+          try {
+            ctx.body = await this.getSystemSettingsInstance();
+          } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)